pyvista/plotting/themes.py

```diff
@@ -2914,6 +2918,28 @@ def lighting_params(self, config: _LightingConfig):  # numpydoc ignore=GL08
             raise TypeError('Configuration type must be `_LightingConfig`.')
         self._lighting_params = config
 
+    @property
+    def logo_file(self) -> Optional[pathlib.Path]:  # numpydoc ignore=RT01
```

Comment:
do we also need to allow for a `str` type?  I see this is used in `add_logo_widget`.

Reply:
The setter will always make it a `pathlib.Path` object or `None`

Reply:
The setter will always make it a pathlib.Path object or None

Reply:
I updated all of this to set the value as a `str` on the theme object to ensure its serializable